app/views/firms/partials/_header.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="firm__header">
+  <%= heading_tag(@firm.name, level: 2, class: 'firm__name') %>
+  <div class="firm__adviser-distance">
+    <%= t('search.result.adviser') %>
+    <%= @firm.closest_adviser %>
+  </div>
+
+  <div class="firm__links">
+    <%= link_to "tel:#{@firm.telephone_number}", class: 'outline-button' do %>
+      <%= svg_icon :phone, title: t('firms.show.telephone_number'), class: 'outline-button__svg-icon'
+      %><%= @firm.telephone_number %>
```

Comment:
Out of interest does this create a weird space when the closing bracket isn't inline like this?